### DISCUSSION TITLE
Define subfigure captions from Python

### DISCUSSION THREAD

**Discussion by ricklupton** [quarto-dev/quarto-cli#10137] [Q&A]
### Description

Multiple figures from code cells can be shown as subfigures (as described in https://quarto.org/docs/get-started/computations/text-editor.html#multiple-figures), but it is not currently possible to define the subfigure captions from the code, only statically in the cell options metadata.

It would be very useful to be able to somehow output metadata for figure layout from the Python code.

**Comment by mcanouil** [comment]
You already can, but that means your Python code needs to emit markdown with the figure syntax, see <https://quarto.org/docs/authoring/figures.html>.

Quarto is unlikely to provide another way to do things especially because it would need to work as well for the other engines.

For short:
- use code cell options
- emit markdown
